[<vsgxchange>]  cannot create ALIAS target "utf8::cpp" because another target   with the same name already exists.

### Operating system

Ubuntu 20.04 LTS

### Compiler

GCC 9.4.0

### Steps to reproduce the behavior

```Shell
// vcpkg.json
{ "name": "",
   "version": "",
  "dependencies": {
      "name": "vsgxchange",
      "default-features": false,
      "features": ["assimp"]
   },
}
```


### Failure logs

CMake Error at /home/user/vcpkg/scripts/buildsystems/vcpkg.cmake:639 (_add_library):
  _add_library cannot create ALIAS target "utf8::cpp" because another target
  with the same name already exists.
Call Stack (most recent call first):
  build/vcpkg_installed/x64-linux/share/utf8cpp/utf8cppConfig.cmake:30 (add_library)
  /home/user/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  /snap/cmake/1329/share/cmake-3.27/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  build/vcpkg_installed/x64-linux/share/assimp/assimpConfig.cmake:38 (find_dependency)
  /home/user/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  /snap/cmake/1329/share/cmake-3.27/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  build/vcpkg_installed/x64-linux/share/vsgXchange/vsgXchangeConfig.cmake:4 (find_dependency)
  /home/user/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  src/core/app/CMakeLists.txt:11 (find_package)

### Additional context

_No response_

I had the same problem and it was solved with this commit. (I have the same ubuntu and gcc version)
But atfter this solution I got a new error:

CMake Error at /home/sascha/develop/vcpkg/scripts/buildsystems/vcpkg.cmake:639 (_add_library):
  _add_library cannot create ALIAS target "utf8::cpp" because target
  "utf8cpp" is imported but not globally visible.
Call Stack (most recent call first):
  /home/sascha/develop/vcpkg/installed/x64-linux/share/utf8cpp/utf8cppConfig.cmake:31 (add_library)
  /home/sascha/develop/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  /usr/share/cmake-3.16/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /home/sascha/develop/vcpkg/installed/x64-linux/share/assimp/assimpConfig.cmake:38 (find_dependency)
  /home/sascha/develop/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  CMakeLists.txt:30 (find_package)

--- a/ports/utfcpp/portfile.cmake
+++ b/ports/utfcpp/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO nemtrif/utfcpp
     REF v${VERSION}
-    SHA512 5135b13a03ee814cb35e04459b2d91b8fbe91cd518a604c41062b4ad42b739fce1acf946b01904309e0edffb874f5e81f69d28afdc8b6f759ef2d675ca0c0db0
+    SHA512 1ce12c8158a2f3bcddec104ceacedaea4031b4c88fc0fa1f1fae8dfa8df81c846861df9d01e8f294d79b9e4ab8c51bd1289f404eed24d07abc760688fee13090
     HEAD_REF master
 )
 

--- a/ports/utfcpp/portfile.cmake
+++ b/ports/utfcpp/portfile.cmake
@@ -1,7 +1,9 @@
+set(VCPKG_BUILD_TYPE release) # header-only
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO nemtrif/utfcpp
-    REF v${VERSION}
+    REF "v${VERSION}"
     SHA512 1ce12c8158a2f3bcddec104ceacedaea4031b4c88fc0fa1f1fae8dfa8df81c846861df9d01e8f294d79b9e4ab8c51bd1289f404eed24d07abc760688fee13090
     HEAD_REF master
 )
@@ -20,9 +22,7 @@ if (VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     vcpkg_cmake_config_fixup(PACKAGE_NAME utf8cpp CONFIG_PATH cmake)
 else()
     vcpkg_cmake_config_fixup(PACKAGE_NAME utf8cpp CONFIG_PATH lib/cmake/utf8cpp)
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
 endif()
-
-# Header only
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug" "${CURRENT_PACKAGES_DIR}/lib")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")

--- a/ports/utfcpp/vcpkg.json
+++ b/ports/utfcpp/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "utfcpp",
-  "version": "3.2.4",
+  "version": "3.2.5",
   "description": "UTF-8 with C++ in a Portable Way",
   "homepage": "https://github.com/nemtrif/utfcpp",
   "license": "BSL-1.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8525,7 +8525,7 @@
       "port-version": 1
     },
     "utfcpp": {
-      "baseline": "3.2.4",
+      "baseline": "3.2.5",
       "port-version": 0
     },
     "utfz": {

--- a/versions/u-/utfcpp.json
+++ b/versions/u-/utfcpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "323b2ffdbd38cf8e756a5ae5714ef2169c0cd6b3",
+      "version": "3.2.5",
+      "port-version": 0
+    },
+    {
       "git-tree": "6c3de7ba47b92f266d5cce0a4f69925af4abda8f",
       "version": "3.2.4",
       "port-version": 0

